apps/jobs/admin.py

```diff
@@ -1,11 +1,9 @@
 from django.contrib import admin
 
 from base.admin import TimeStampedAdmin
-
 from .models import Submission
 
 
-@admin.register(Submission)
```

Comment:
@AnshulBasia There is no need to make changes in this file.  you just need to delete the `submissions` app and then remove from installed apps.